test/compilable/test21177.d

```diff
@@ -0,0 +1,26 @@
+// https://issues.dlang.org/show_bug.cgi?id=21177
+/*
+DISABLED: win
+TEST_OUTPUT:
+---
+compilable/test21177.d(19): Deprecation: more format specifiers than 0 arguments
+---
+*/
+
+import core.stdc.stdio;
+
+void main()
+{
+    version (CRuntime_Glibc)
```

Comment:
You might want to add a `#line` to keep the test output stable for any future additions.